### DISCUSSION TITLE
How can I change the language of DateTimePicker content

### DISCUSSION THREAD

**Discussion by IgorGribowsky** [HandyOrg/HandyControl#631] [Q&A] [answered]
I downloaded your package from NuGet and am trying to use it for my application. How can i changer the conrol content language to English? I've tried several ways, but they don't work. Please, help!

![image](https://user-images.githubusercontent.com/47255467/105182632-51003480-5b3e-11eb-8b71-4c45c8b24868.png)

I tried doing this:
` <hc:DateTimePicker Language="en" SelectedDateTime="{Binding StartDate, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" />`

and this:
![image](https://user-images.githubusercontent.com/47255467/105182990-bd7b3380-5b3e-11eb-9101-9806c6a4354a.png)

**Comment by ghost1372** [comment, marked as answer]
you should use
`ConfigHelper.Instance.SetLang("en");
`
make sure en folder exist next to the exe
![Untitled](https://user-images.githubusercontent.com/9213496/105187338-eb18aa80-5b47-11eb-9fd8-789357e43226.png)

**Reply by ms-somanna** [reply]
Should we create an empty en.resx in the en folder? If not what are we supposed to put in the resx file?

**Reply by ghost1372** [reply]
@SwaroopSomanna  No, Language folders are created automatically after the project is built and contain a DLL file
![1](https://user-images.githubusercontent.com/9213496/105285984-af72f480-5bca-11eb-984f-2573bf889bb2.png)
![2](https://user-images.githubusercontent.com/9213496/105285990-b39f1200-5bca-11eb-8446-caff51c5523b.png)

**Comment by linshuchang** [comment]
Why do I speak half Chinese and half English？
![PixPin_2024-11-15_14-34-46](https://github.com/user-attachments/assets/813d5f03-c6f3-4706-875e-4a80daa9d98c)

**Reply by linshuchang** [reply]
![PixPin_2024-11-15_18-08-13](https://github.com/user-attachments/assets/3fc536bc-c5d4-482c-87bc-21a4b74e802f)
